cypress.json

```diff
@@ -12,7 +12,7 @@
   "failOnStatusCode": false,
   "ignoreTestFiles": [
     "**/examples/*",
-    "**/!(*.spec).js"
+    "**/*.spec.js"
```

Comment:
Why did this change?

Reply:
because it was ignoring all `.js` files

Reply:
Previously it would ignore any JS file that is not a spec (so spec was included). Now spec is being excluded. Why?

Reply:
because we will use `***.test.js` as file name to run and just ignore `unit` tests (`***.spec.js`)

Reply:
But why are we ignoring `*.spec.js` previously it was explicitly included.